src/status_im/contexts/chat/messenger/composer/actions/view.cljs

```diff
@@ -62,7 +66,7 @@
      [quo/button
       {:icon-only?          true
        :size                32
-       :customization-color customization-color
+       :customization-color (or contact-customization-color color customization-color)
```

Comment:
I'm very confused by these variable names, they all seem the same. Can we change the names to provide a bit more of clarity why it's done this way?